lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php

```diff
@@ -45,6 +47,11 @@ class ObjectHydrator extends AbstractHydrator
     private $identifierMap = [];
 
     /**
+     * @var object[] indexed by oid
+     */
+    private $createdEntities = [];
```

Comment:
This could probably need a rename - the idea of this field is that it tracks entities that we can write to in the scope of this particular hydrator.

Reply:
`$trackedWritableEntities`?